src/main/java/com/minecolonies/coremod/compatibility/jei/JEIPlugin.java

```diff
@@ -0,0 +1,46 @@
+package com.minecolonies.coremod.compatibility.jei;
+
+import com.minecolonies.coremod.compatibility.jei.transer.PrivateCraftingTeachingTransferHandler;
+import mezz.jei.api.IJeiRuntime;
+import mezz.jei.api.IModPlugin;
+import mezz.jei.api.IModRegistry;
+import mezz.jei.api.ISubtypeRegistry;
+import mezz.jei.api.ingredients.IModIngredientRegistration;
+import mezz.jei.api.recipe.IRecipeCategoryRegistration;
+import mezz.jei.api.recipe.VanillaRecipeCategoryUid;
+import mezz.jei.api.recipe.transfer.IRecipeTransferRegistry;
+
+@mezz.jei.api.JEIPlugin
+public class JEIPlugin implements IModPlugin
+{
+    @Override
+    public void registerItemSubtypes(final ISubtypeRegistry iSubtypeRegistry)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1186)